CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## 0.11.3 - 2022-11-28
 
+### Changed
+
+- Updated `axum` to v0.6.0.
```

Comment:
this isn't right, it should go on the unreleased section

Reply:
I've just noticed that I branched from `master` and not from `dev`. Gonna fix that and the changelog in a moment.